src/main/java/com/azure/cosmos/examples/changefeedpull/Resources.java

```diff
@@ -0,0 +1,184 @@
+package com.azure.cosmos.examples.changefeedpull;
+
+import com.azure.cosmos.implementation.guava25.collect.Multimap;
+import com.azure.cosmos.CosmosAsyncClient;
+import com.azure.cosmos.CosmosAsyncContainer;
+import com.azure.cosmos.CosmosAsyncDatabase;
+import com.azure.cosmos.implementation.guava25.collect.ArrayListMultimap;
+import com.azure.cosmos.models.ChangeFeedPolicy;
+import com.azure.cosmos.models.CosmosContainerProperties;
+import com.azure.cosmos.models.CosmosContainerResponse;
+import com.azure.cosmos.models.CosmosDatabaseResponse;
+import com.azure.cosmos.models.CosmosItemResponse;
+import com.azure.cosmos.models.PartitionKey;
+import com.azure.cosmos.models.ThroughputProperties;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.UUID;
+import java.util.function.Function;
+
+public class Resources {
+    CosmosAsyncContainer container;
+    CosmosAsyncDatabase database;
+    CosmosAsyncClient clientAsync;
+    public String DATABASE_NAME;
+    public String COLLECTION_NAME;
+    private final Multimap<String, ObjectNode> partitionKeyToDocuments = ArrayListMultimap.create();
```

Comment:
Although it is being used in insert, the map is actually redundant here because update is not being used (I am leaving it here for future when this sample can be updated to include ffcf). Either way, it really needs multi-map so we can have multiple values per key (an easier way to model partition key when retrieving docs - see update method). That said, I've replaced with ArrayListValuedHashMap from Apache Commons so we don't use Cosmos implementation packages etc.